jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -130,38 +130,44 @@ public interface Request extends Attributes, Content.Source
     String getId();
 
     /**
-     * @return the {@link Components} to be used with this request.
+     * @return the {@link Components} to be used with this {@code Request}.
      */
     Components getComponents();
 
     /**
-     * @return the {@code ConnectionMetaData} associated to this request
+     * @return the {@code ConnectionMetaData} associated to this {@code Request}
      */
     ConnectionMetaData getConnectionMetaData();
 
     /**
-     * @return the HTTP method of this request
+     * @return the HTTP method of this {@code Request}
      */
     String getMethod();
 
     /**
-     * @return the HTTP URI of this request
+     * @return the HTTP URI of this {@code Request}
      * @see #getContextPath(Request)
      * @see #getPathInContext(Request)
      */
     HttpURI getHttpURI();
 
     /**
-     * @return the {@code Context} associated with this {@code Request}
+     * Get the {@link Context} associated with this {@code Request}.
+     * <p>Note that a {@code Request} should always have an associated {@link Context} since if the
+     * {@code Request} is not being handled by a {@link org.eclipse.jetty.server.handler.ContextHandler} then
+     * the {@link org.eclipse.jetty.server.Server.ServerContext} instance will be returned.
+     * @return the {@code Context} associated with this {@code Request}. Never {@code null}.
```

Comment:
No need to specify all subclasses -- just say it's never null.

Reply:
This does not specify "all the subclasses", it just give a very important specific example.